### PR TITLE
Set path to docker command dynamically in rbe_autoconfig

### PR DESCRIPTION
**Pull request by nlopezgi**
Also make copy_resources the default

### CODE REVIEW

File: rules/rbe_repo.bzl [hunk 1]
```diff
@@ -348,16 +350,25 @@ def _print_exec_results(prefix, exec_result, fail_on_error = False, args = None)
...
-        fail("Cannot run rbe_autoconfig as 'docker' was not found on the path.")
-    if ctx.execute(["docker", "ps"]).return_code != 0:
-        fail("Cannot run rbe_autoconfig as running 'docker ps' returned a " +
-             "non 0 exit code, please check you have permissions to run docker.")
+    docker_tool_path = ctx.os.environ.get(_DOCKER_PATH, None)
+    if not docker_tool_path:
+        docker_tool_path = ctx.which("docker")
+    if not docker_tool_path:
+        fail("Cannot run rbe_autoconfig as 'docker' was not found on the " +
+             "path and environment variable _DOCKER_PATH was not set.")
```
Comment (smukherj1) [inline rules/rbe_repo.bzl]: "_DOCKER_PATH" should be "DOCKER_PATH"

Comment (nlopezgi) [reply rules/rbe_repo.bzl]: fixed

**Pull request merged**